src/plugins/openaq/__init__.py

```
@@ -0,0 +1,6 @@
+from .datasource import OpenAQDataSource
+from ..import PluginRegistry
+
+
+def register_plugin(registry: PluginRegistry) -> None:
+    registry.register('openaq', OpenAQDataSource)
```

Pylint (3.3.7)

[convention] Final newline missing

(C0304)